lib/simulator-xcode-9.js

```diff
@@ -150,6 +150,29 @@ class SimulatorXcode9 extends SimulatorXcode8 {
           `'${prefs.SimulatorWindowLastScale}' is assigned instead.`);
       }
     }
+
+    if (!_.isUndefined(prefs.SimulatorWindowCenter)) {
+      const verificationPattern = /\{\-?\d+(\.\d+)?,\-?\d+(\.\d+)?\}/;
```

Comment:
Can you document this regex in some way? Perhaps in https://regex101.com/ or something?

Reply:
would it be ok https://regex101.com/r/2ZXOij/1 ?

Reply:
Awesome!

This points out, though, that the capture groups are unnecessary, since nothing is being done with what is captured (`/\{\-?\d+\.\d+?,\-?\d+\.\d+?\}/` would suffice). I don't think it matters here because the efficiency is not really at issue for such a short search. But in general capturing text is much less performant in a regex.

Reply:
I just wanted to make sure the expression like this is not possible
`{1., 2.}`
Would it be possible without groups?

Reply:
Ah. Yes, you need them for that. Can you put the various variations in the linked regex test (you'll need to add the `m` flag so it is multiline there?
```
# should match
{100.0,100.0}
{100,100}
{100.0,100}

# should not match
{100.0, 100.0}
{100.0,100.}
```

Reply:
I thought a unit test would be enough for this purpose. But yeah, it's easy to add